python/timeplus/query.py

```diff
@@ -16,21 +16,21 @@ def __init__(self, env):
         )
         self._create_response = None
         self._id = None
-        self._batching_pilicy = None
+        self._batching_policy = None
 
     def sql(self, query):
         self._sql = query
         return self
 
-    def batching_pilicy(self, count, time_ms):
-        self._batching_pilicy = swagger_client.models.BatchingPolicy(
+    def batching_policy(self, count, time_ms):
```

Comment:
maybe add doc for this function, when this method should be called, any benefit or drawback?